js/script.js

```diff
@@ -11,4 +11,217 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+  },
+  {
+    id: 'bg-pro3',
+    name: 'Website Protfolio',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: '../img/Img Placeholder1-1.svg',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    seeProject: 'See Project',
+    linkLive: 'See live',
+    linkSource: 'See source',
+  },
+  {
+    id: 'bg-pro4',
+    name: 'Professional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: '../img/Img Placeholder1-3.svg',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    seeProject: 'See Project',
+    linkLive: 'See live',
+    linkSource: 'See source',
+  },
+  {
+    id: 'bg-pro5',
+    name: 'Data Dashboard Healthcare',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: '../img/Img Placeholder1-2.svg',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    seeProject: 'See Project',
+    linkLive: 'See live',
+    linkSource: 'See source',
+  },
+  {
+    id: 'bg-pro6',
+    name: 'Website Protfolio',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: '../img/Img Placeholder1-1.svg',
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    seeProject: 'See Project',
+    linkLive: 'See live',
+    linkSource: 'See source',
+  },
+];
+// pop-up section
+const seeProject = document.querySelectorAll('.work-button');
+const popMenu = document.querySelector('.seePopup');
+
+seeProject.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    document.body.classList.toggle('no-scroll');
+    popMenu.classList.add('act');
+    popMenu.innerHTML = `    
+    <section id="popup" class="popup">
+        <div id="popup-title">
+          <h3 class="popup-heading flex">${allProjects[index].name}<h3>
+            <span class="close-btn">&#x2715;</span>
+            <div class="flex
+            flex-start">
+              <p class="flex-item">${allProjects[index].technologies[0]}
+              </p>
+              <p class="flex-item">${allProjects[index].technologies[1]}
+              </p>
+              <p class="flex-item">${allProjects[index].technologies[2]}
+              </p>
+          </div>
+        </div>
+          <div class="popup-card">
+             <div id="popup-image"><img src="${allProjects[index].featuredImage}" alt="something here">
+              </div>
+              <div class="flex flex-column">
+                <p class="popup-details">
+                ${allProjects[index].description}
+                </p>
+                <div class="flex popup-textbtn">
+                  <button id= "see-live" 
+                  class="work-button"><a href="https://fpsapc.github.io/MicroverseStudentProject1/">${allProjects[index].linkLive}</a>
+                  <img src="../img/live.png" alt="live icon">
```

Comment:
- Same here, kindly check the path of your `src` image

![image](https://user-images.githubusercontent.com/93630700/223752365-272b75cd-5519-4638-982f-8347c9fae69b.png)

Reply:
Thanks for your kind review
@BregornOriginal now got the point and change all paths as you suggests.